lib/features/children/family_history/family_history_logic/family_history_repository.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:givt_app/core/network/api_service.dart';
+import 'package:givt_app/features/children/family_history/models/allowance.dart';
+import 'package:givt_app/features/children/family_history/models/child_donation.dart';
+import 'package:givt_app/features/children/family_history/models/history_item.dart';
+
+mixin FamilyHistoryRepository {
```

Comment:
just wondering why do you combine both cubit and repositories folders into one _logic? how is it consistent with other features folder structure?